commands/member/cabbages.js

```diff
@@ -1,13 +1,152 @@
-const { SlashCommandBuilder } = require('discord.js');
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+const { getDiscordClient } = require('../../discord');
+const {
+    getCabbageBreakdown,
+    cabbagesUntilNext,
+} = require('../../helpers/calculateCabbages');
 const models = require('../../models');
 
+const findEmoji = (client, name) => {
+    emoji = client.emojis.cache.find(
+        (emoji) => emoji.name.toLowerCase() === name.toLowerCase()
+    );
+    return emoji || '';
+};
+
+const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
+};
+
+const inlinefield = (name, value) => ({
+    name: name || ' ',
+    value: value || ' ',
+    inline: true,
+});
+
+const cabbageEmbed = (member, memberData) => {
+    const client = getDiscordClient();
+    const { accountProgression: account } = memberData;
+    // Generate all neccesary info
+    const { nickname } = member;
+    const checkmark = findEmoji(client, 'check');
+    const rankEmojiName = `${memberData.currentRank
+        .toLowerCase()
+        .replace(/ /g, '')}Gem`;
+    const rankEmoji = findEmoji(client, rankEmojiName);
+    const cabbages = Math.floor(memberData.currentCabbages);
+    const cabbageBreakdown = getCabbageBreakdown(memberData);
+    const untilNextTier = cabbagesUntilNext(cabbages);
+    const date_unix = Math.floor(Date.parse(memberData.updatedAt) / 1000);
+    // Generate embed fields
+    const achievementText = [];
+    const statusText = [];
+    const cabbagesText = [];
+    let currEmoji;
```

Comment:
Better to remove this variable here and just redefine it in every if-statement, since they have no correlation to each other anyway and could lead to possible mistakes further down the line

Reply:
I'll close this requested change for now but I mentioned it in issue #23